third_party/terraform/tests/resource_google_project_organization_policy_test.go

```diff
@@ -16,12 +16,13 @@ import (
 // avoid race conditions and aborted operations.
 func TestAccProjectOrganizationPolicy(t *testing.T) {
 	testCases := map[string]func(t *testing.T){
-		"boolean":        testAccProjectOrganizationPolicy_boolean,
-		"list_allowAll":  testAccProjectOrganizationPolicy_list_allowAll,
-		"list_allowSome": testAccProjectOrganizationPolicy_list_allowSome,
-		"list_denySome":  testAccProjectOrganizationPolicy_list_denySome,
-		"list_update":    testAccProjectOrganizationPolicy_list_update,
-		"restore_policy": testAccProjectOrganizationPolicy_restore_defaultTrue,
+		// "boolean":        testAccProjectOrganizationPolicy_boolean,
```

Comment:
These should probably be uncommented, right?

Reply:
oops, thanks.